docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -35,7 +78,21 @@ Most of this article is about how to use the <xref:System.Text.Json.JsonSerializ
 
 During deserialization, `Newtonsoft.Json` does case-insensitive property name matching by default. The <xref:System.Text.Json> default is case-sensitive, which gives better performance since it's doing an exact match. For information about how to do case-insensitive matching, see [Case-insensitive property matching](system-text-json-how-to.md#case-insensitive-property-matching).
 
-If you're using `System.Text.Json` indirectly by using ASP.NET Core, you don't need to do anything to get behavior like `Newtonsoft.Json`. ASP.NET Core specifies the settings for [camel-casing property names](system-text-json-how-to.md#use-camel-case-for-all-json-property-names) and case-insensitive matching when it uses `System.Text.Json`.
+If you're using `System.Text.Json` indirectly by using ASP.NET Core, you don't need to do anything to get behavior like `Newtonsoft.Json`. ASP.NET Core specifies case-insensitive matching when it uses `System.Text.Json`.
+
+### Camel-case property names
+
+During serialization, `Newtonsoft.Json` converts property names to camel case by default. The <xref:System.Text.Json> default is to keep property name casing the same. For information about how to camel-case property names, see [Use camel case for all JSON property names](system-text-json-how-to.md#use-camel-case-for-all-json-property-names).
```

Comment:
> During serialization, `Newtonsoft.Json` converts property names to camel case by default.

This isn't true. Newtonsoft has the same default as S.T.J. It does exact property name match on serialization. On deserialization, since it is case-insensitive, it will allow any casing to match with the property name.

```C#
public class CasingOutput
{
    public int PascalCase { get; set; }
    public int camelCase { get; set; }
    public int ranDoMCAse { get; set; }
}

var casing = new CasingOutput
{
    PascalCase = 1,
    camelCase = 2,
    ranDoMCAse = 3
};

// {"PascalCase":1,"camelCase":2,"ranDoMCAse":3}
Console.WriteLine(JsonConvert.SerializeObject(casing));
```

Should we instead merge the case insensitive and camel case sections into one and just refer to them individually in the table (like we do for #conditionally-ignore-a-property)? And then update the h3 title?

The only problem with doing that is this h2 is about differences in **default** behavior which doesn't differ here. Maybe we just link to the camel case doc directly in the table?

So the table entries would be:
Case-insensitive deserialization by default -> [PropertyNameCaseInsensitive global setting]#case-insensitive-deserialization
Camel-case property names using CamelCaseNamingStrategy -> [PropertyNamingPolicy global setting] system-text-json-how-to.md#use-camel-case-for-all-json-property-names

Up to you. (or we could remove this section all together)

Reply:
I went the link-directly-to-the-other-doc route, and since we had the same problem with NullValueHandling (not a difference in default behavior), I did that with it too and removed the h3 section from this doc.